Sources/Amplitude/Amplitude.m

```diff
@@ -255,21 +258,6 @@ - (instancetype)initWithInstanceName:(NSString *)instanceName {
                 AMPLITUDE_LOG(@"Loaded from %@", _propertyListPath);
             }
 
-            // update database if necessary
-            int oldDBVersion = 1;
-            NSNumber *oldDBVersionSaved = [self->_propertyList objectForKey:DATABASE_VERSION];
```

Comment:
Is this old migration? Just curious is there any plan about the `DATABASE_VERSION`?

Reply:
I thought we were completely removing the old migration. I also removed a very old (5 years old) migration from the Android DB.